minargon/static/css/minargon.css

```diff
@@ -84,19 +84,25 @@ div.alert-holder {
 
```

Comment:
What do these changes in css do? What are they for?

Reply:
These changes fix the white background in the dropdown menus. At least, this was the solution